src/getAppSyncConfig.js

```diff
@@ -197,8 +197,8 @@ export default function getAppSyncConfig(context, appSyncConfig) {
 
       const defaultTemplatePrefix = getDefaultTemplatePrefix(template);
 
-      const requestTemplate = request || `${defaultTemplatePrefix}.request.vtl`;
-      const responseTemplate = response || `${defaultTemplatePrefix}.response.vtl`;
+      const requestTemplate = request || cfg.defaultMappingTemplates.request || `${defaultTemplatePrefix}.request.vtl`;
+      const responseTemplate = response || cfg.defaultMappingTemplates.response || `${defaultTemplatePrefix}.response.vtl`;
```

Comment:
@abdala Thanks for fixing this!!
There is one more thing I forgot about when reviewing yesterday:
- `request`, `response`, `cfg.defaultMappingTemplates.reques` and `cfg.defaultMappingTemplates.response` can contain `false`!! (when using direct lambdas).

This would not work as it is (and definitely did not work before either).
When the value is present and `false`, it should be honored.
check [here](https://github.com/sid88in/serverless-appsync-plugin/blob/master/src/index.js#L881-L884)

There is only one more thing to take into consideration here: This plugin (aws amplify-cli) don't support direct lambdas yet!!
We should still honor `false` values, but we should detect them and probably throw an Exception to warn users.

> Sorry, this plugin does not support Direct Lambda resolvers yet.

I still need to find some time to implement Direct Lambdas properly, either here or open a PR in https://github.com/aws-amplify/amplify-cli

Thanks again

Reply:
If I got it correctly this is implementation, right?

Reply:
Huummm, no....

Reply:
I neeeeeedd teeesstt! hahahah

Reply:
If we check for this before I think it is fine :)

Reply:
@abdala Correct implementation should be

```js
const requestTemplate = !isNil(request)
	? request
	: !isNil(cfg.defaultMappingTemplates.request)
		? cfg.defaultMappingTemplates.request
		: `${defaultTemplatePrefix}.request.vtl`;

if (requestTemplate === false) {
	throw new Error('Sorry, this plugin does not support Direct Lambda resolvers yet.');
}
```

default template can be `false ` too.

\* [isNil](https://lodash.com/docs/4.17.15#isNil)

Reply:
I see. Tks!